gslb/nodes/avi_model_nodes.go

```diff
@@ -129,9 +130,14 @@ type PathHealthMonitorDetails struct {
 	Name            string
 	IngressProtocol string
 	Path            string
+	RequestHeader   string
+	ResponeCodes    []string
```

Comment:
I can't see `RequestHeader` and `ResponeCodes` being used anywhere. Can we remove it?

Reply:
I have removed the two unused variables.